music21/chord/__init__.py

```diff
@@ -4885,7 +4885,7 @@ def pitchedCommonName(self) -> str:
             return f'{rootName}-{nameStr}'
 
     @property
-    def pitches(self) -> Tuple[pitch.Pitch]:
+    def pitches(self) -> Tuple[Union[pitch.Pitch, pitch.Unpitched]]:
```

Comment:
> Notationally, unpitched notes and pitched notes are very similar, but from an analytical perspective, having some notes whose .ps is meaningless, or Chords where "isTriad" doesn't mean that it sounds like a triad, is going to cause too many problems.

Yep, as I started overridding more and more properties not return unhelpful things it was clear how confusing this could get. Makes sense.

> I think that the note.Unpitched object needs to be restored OR something similar moved to percussion.py -- we probably need to insert some layer between NotRest (which I wish I had called something like Stemmable, but it is too late now) and Chord -- ChordBase or something, from which a PercussionChord or something like that appears.

I guess I would still appreciate a design where the underlying NotRest could be edited in place to be made unpitched. Maybe we can't. But worth thinking about:


```
n = note.Note()
n.ps = 60
n.makeUnpitched(inPlace=True)
n.ps = NaN  # or however we access the fact that this no longer pitched.
n.pitch.midi = 60
n.addLyric("spoken word")
```

Reply:
I think that better would be something like a classmethod on wherever Unpitched ends up going something like:

```
@classmethod
def unpitchedFromNote(cls, n: note.Note):
      unpitch = Unpitched()
      ... merge attributes ...
      return unpitch
```

I like moving rarer items to more rarely used modules, so that it's easier when someone does `dir(note.Note)` to get a handle on what a note normally does.